resource-definitions/terraform-driver/custom-git-config/example-def.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: entity.humanitec.io/v1b1
+metadata:
+  id: example-git-config
+entity:
+  criteria: {}
+  driver_inputs:
+    values:
+      files:
+        .gitconfig: |
+          [url "https://github.com/Invicton-Labs/"]
```

Comment:
Is this the name of a prospect? If so, we should adjust the URL.

Reply:
No. It is a random publicly available terraform provider